go/consensus/tendermint/full/full.go

```diff
@@ -774,6 +774,7 @@ func (t *fullService) GetStatus(ctx context.Context) (*consensusAPI.Status, erro
 	status.ChainContext = t.genesis.ChainContext()
 	status.GenesisHeight = t.genesis.Height
 	if t.started() {
+		status.Status = "Ready"
```

Comment:
Shouldn't this check if `t.syncedCh` is closed? Because a node can be started and still syncing?

Reply:
Yeah of course.